src/sql.hpp

```diff
@@ -130,6 +130,9 @@ namespace nogdb {
             nogdb::Record toBaseRecord() const;
 
         private:
+
+            friend nogdb::Record;
```

Comment:
Why `nogdb::sql_parser::Record` class is friends of `nogdb::Record`? I think one way relationship is enough.